app/controllers/lti_deployment_controller.rb

```diff
@@ -90,12 +46,11 @@ def redirect_login
       render 'shared/http_status', locals: { code: '422', message: I18n.t('lti.config_error') }, layout: false
       return
     end
-
     referrer_uri = URI(request.referer)
     # Get canvas JWK set
```

Comment:
There are a few different comments in this file that still refer to "canvas", and should now refer to "LMS" (or whatever term the LTI standard uses, like "platform" or something).